base/compiler/ssair/passes.jl

```diff
@@ -594,7 +597,7 @@ end
 
 struct LiftedPhi
     ssa::AnySSAValue
-    node::PhiNode
+    node::Union{PhiNode,Expr}
```

Comment:
Since `Expr` is a somewhat arbitrary type, it might be better to create a new struct, let's say `IfElseCall`, and include it here? This would make it explicit that the lifted value is a call to `Core.ifelse`. Additionally, it might be better to rename `LiftedPhi` to `LiftedNode`.